thunderbird configuration ```0344``` (#511)
Start9Labs/documentation: site/source/guides/misc-guides/lan-thunderbird.rst

.. _lan-thunderbird:

=========================
Thunderbird Configuration
=========================
.. caution:: You will first need to complete :ref:`device-guides` for your client before continuing.  Take special note of the Thunderbird-specific instructions there.

Trusting Your Start9 CA in Thunderbird
======================================

#. Open Thunderbird and do the following:

>     a. Select the hamburger menu bar and click **Settings** from the resulting drop-down menu.

Reviewer: ```suggestion
    a. Select the hamburger menu and click **Settings** from the resulting drop-down menu.
```

  
    .. figure:: /_static/images/nextcloud/thunderbird-ca-1.png
        :width: 60%
        :alt: Thunderbird settings

    b. Search for the term ``config`` in the search bar and click **Config Editor...**.

    .. figure:: /_static/images/nextcloud/thunderbird-ca-2.png
        :width: 60%
        :alt: Thunderbird config editor

#. Search for ``security.enterprise_roots.enabled``, and click the toggle button to ``true``.

    .. figure:: /_static/images/nextcloud/thunderbird-ca-3.png
        :width: 60%
        :alt: Thunderbird enterprise_roots settings

Configure Tor in Thunderbird
============================

> .. note:: This only applies if you have **NOT** enabled sytem-wide Automatic proxy configuration. 

Reviewer: ```suggestion
.. note:: This is not required if you have enabled a sytem-wide proxy configuration. 
```

#. Open Thunderbird and do the following:

    a. Select the hamburger menu bar and click **Settings** from the resulting drop-down menu.

    .. figure:: /_static/images/nextcloud/thunderbird-ca-1.png
        :width: 60%
        :alt: Thunderbird settings

    b. Search for the term ``proxy`` in the search bar and click **Settings...**:

    .. figure:: /_static/images/nextcloud/thunderbird-proxy-1.png
        :width: 60%
        :alt: Thunderbird proxy settings

#. Choose **Automatic proxy configuration URL** and paste the appropriate URL:

>     .. caution:: This step assumes you saved proxy.pac file as per :ref:`device-guides`. If you saved this file in different location please provide correct path.

Reviewer: ```suggestion
    .. caution:: This step assumes you saved the proxy.pac file as suggested in the :ref:`device-guides`. If you saved this file in different location please provide correct the path.
```

>    - For Linux: ``file:///home/$YOUR_USERNAME/proxy.pac``

Reviewer: ```suggestion
   - For Linux: ``file:///etc/tor/proxy.pac``
```

This is the file location in device guides, not the user folder
